metriq-api/service/userService.js

```diff
@@ -286,6 +273,26 @@ class UserService extends ModelService {
     return { success: true, body: await this.getSanitized(user.id) }
   }
 
+  async update(userId, reqBody) {
```

Comment:
It's probably because you're not putting the user ID in the `id` property of the request on the front end.

Come to think of it, so long as this works, it's acceptable. If you're getting the user ID from the route rather than the request, then you're probably getting it from the requesting user's token, actually. Either way, "spoofing" is prevented, but this might be even better.

This is acceptable, so long as you're using the original controller route I asked you to use. You can change the service method signature.

Reply:
Gotcha. Okay, so from what I understand this should be good to go in its current form (let me know if I'm wrong). 

If so, I can merge this one first and then the companion `metriq-app` diff.

Reply:
Please just remove the new redundant route mapping, first, but _pending that_, I'll consider this approved upon agreement to make that change.

It's the submission workflow that's going to be more important for the "submission party," tomorrow, but it'd be good to have this in, too. Not a big deal either way. Sorry for the amount of time it took review this, but I've had many priorities pulling me different directions, including the submission workflow that's ready for review.